tools/rosmsg/src/rosmsg/__init__.py

```diff
@@ -742,7 +743,9 @@ def rosmsgmain(mode=MODE_MSG):
 
         command = sys.argv[1]
         if command == 'show':
-            sys.exit(rosmsg_cmd_show(ext, full))
+            sys.exit(rosmsg_cmd_show(ext, full, 'show'))
+        elif command == 'info':
+            sys.exit(rosmsg_cmd_show(ext, full, 'info'))
```

Comment:
@javierdiazp Can you please update the code with the suggestion from @mikepurvis